airbyte-integrations/connectors/source-mssql/src/test/java/io/airbyte/integrations/source/mssql/MssqlSourceTest.java

```diff
@@ -48,12 +48,16 @@
 
 class MssqlSourceTest {
 
-  private static final String STREAM_NAME = "dbo.id_and_name";
+  private static final String STREAM_NAMESPACE = "dbo";
+  private static final String TABLE_NAME = "id_and_name";
+  private static final String STREAM_NAME = STREAM_NAMESPACE + "." + TABLE_NAME;
```

Comment:
shouldn't the stream name just be tablename?

Reply:
The previous STREAM_NAME was "dbo.id_and_name"

We didn't introduce big changes on how things work so the stream name is still the same as before (but now it is also just "parsed" and split in two in the new AirbyteStreamName struct)